skimage/transform/tests/test_distance_tranform.py

```diff
@@ -0,0 +1,17 @@
+from skimage.transform.distance_transform import generalized_distance_transform as md

```

Comment:
This file is much nicer now, @leGIT-bot!

Can you remove the alias to `md`, which makes the below very unreadable, and also add tests for `abs` as the distance function?